### PR TITLE
add a mips assembly repository

### PR DESCRIPTION
**Pull request by zakarialaoui10**

### CODE REVIEW

**Review by yezz123 — APPROVED** [review]
LGTM! 👌🏻 just delete this one https://github.com/DevC-Casa/awesome-morocco/pull/108

**Review by zakarialaoui10 — COMMENTED** [review]
update on docs

File: readme.md [hunk 1]
```diff
@@ -273,6 +273,7 @@
 - [CSE](https://github.com/bilnash/CSE) - R package for Casablanca Stock Exchange Data
 - [libCplus](https://github.com/BnademOverflow/libCplus) - Wonderful library with lots of useful functions, algorithms and data structures in C. by [@BnademOverflow](https://github.com/BnademOverflow)
 - [vscode-big-sur-icons](https://github.com/mohouyizme/vscode-big-sur-icons) - Icons for VSCode based on macOS Big Sur design.
+- [HIGH TO LOW](https://github.com/zakarialaoui10/HIGH-TO-LOW)-codes in C and their equivalence in MIPS Assembly by [@zakarialaoui10](https://github.com/zakarialaoui10)
```
Comment (JefferyHus) [inline readme.md:276]: If you can add white space around `-codes` and capitalize the C. Otherwise LGTM

**Review by yezz123 — APPROVED** [review]
Now everything is Good, @JefferyHus +1

**Pull request merged**